cms/djangoapps/contentstore/tests/test_tasks.py

```diff
@@ -277,17 +283,62 @@ def test_check_broken_links_stores_broken_and_locked_urls(
         ### Check that _save_broken_links_file was called with the correct arguments
         mock_save_broken_links_file.assert_called_once_with(mock_user_task_artifact.return_value, mock.ANY)
 
+    @pytest.mark.skip(reason="This test is not yet implemented")
     def test_user_does_not_exist_raises_exception(self):
-        raise NotImplementedError
+        assert True
 
+    @pytest.mark.skip(reason="This test is not yet implemented")
     def test_no_course_access_raises_exception(self):
-        raise NotImplementedError
+        assert True
 
     def test_hash_tags_stripped_from_url_lists(self):
-        raise NotImplementedError
+        NUM_HASH_TAG_LINES = 2
+        url_list = '''
+        href='#'                        # 1 of 2 lines that will be stripped
+        href='http://google.com'
+        src='#'                         # 2 of 2 lines that will be stripped
+        href='https://microsoft.com'
+        src="/static/resource_name"
+        '''
 
-    def test_urls_out_count_equals_urls_in_count_when_no_hashtags(self):
-        raise NotImplementedError
+        original_lines = len(url_list.splitlines()) - 2 # Correct for the two carriage returns surrounding the ''' marks
+        processed_url_list = _get_urls(url_list)
+        processed_lines = len(processed_url_list)
+
+        assert processed_lines == original_lines - NUM_HASH_TAG_LINES, \
+            f'Processed URL list lines = {processed_lines}; expected {original_lines - 2}'
+
+    # TODO - Document here what counts as a legitimate URL & modify test accordingly
+    @pytest.mark.skip(reason="Valid URL format to be nailed down")
```

Comment:
Instead of skipping, could you just remove the test case? I think we got enough tests for now.